eslint-config/readme.md

```diff
@@ -0,0 +1,20 @@
+# @mate-academy/eslint-config
+
+#### A simplified eslint-config based on [standard](https://standardjs.com/rules.html) rules
```

Comment:
Let's point to our version: https://github.com/mate-academy/style-guides/blob/master/javascript-standard-modified.md